internal/impl/devbox.go

```diff
@@ -942,6 +955,23 @@ func (d *Devbox) packages() []string {
 	return d.cfg.Packages
 }
 
+func (d *Devbox) findPackageByName(name string) (string, error) {
+	results := []string{}
+	for _, pkg := range d.cfg.Packages {
+		i := nix.InputFromString(pkg, d.lockfile)
+		if i.String() == name || i.CanonicalName() == name {
+			results = append(results, pkg)
+		}
+	}
+	if len(results) > 1 {
+		return "", usererr.New(
+			"found multiple packages with name %s: %s", name, results)
```

Comment:
how would the user proceed in this scenario? Can we print additional information about the packages and suggest an alternative command argument?

🤔 this function is kinda generic, so hard to give corrective suggestions here. 

So, maybe the above advice could happen in one of the callers of this function where the user-scenario is more clear.

Reply:
You need to provide exact match. So if you have `go@1.20` and `go@1.19` and you try to update `go` you will get this error.

I don't disagree that the error is a bit terse, but this is a minor edge case, generally users will not install multiple versions of same package. I don't think we need special handling?

I'll change to `"found multiple packages with name %s: %s. Please specify version",`